ansible/roles/letsencrypt/tasks/legacy.yml

```diff
@@ -0,0 +1,44 @@
+---
+  - name: Install certbot dependencies
+    become: true
+    apt:
+      pkg: "{{item}}"
+      state: present
+    with_items:
+      - augeas-lenses
+      - ca-certificates
+      - dialog
+      - gcc
+      - libaugeas0
+      - libffi-dev
+      - libpython-dev
+      - libpython2.7-dev
+      - libssl-dev
+      - python
+      - python-dev
+      - python-setuptools
+      - python-virtualenv
+      - python2.7-dev
```

Comment:
do you need python 2.7?

Reply:
python2.7 is a dependency of python2.7-dev so it would get installed but i will make it explicit